docs/dev-process/resx-files.md

```diff
@@ -0,0 +1,24 @@
+# Resources
+
+Resources are `.resx` files with string values that we use for error messages and such.
+They live in `src\Engine\resources` and `src\Rules\resources` folders.
+
+At the moment `dotnet cli` doesn't support automatically generating C# bindings (strongly typed resource files).
+Script analyzer resources are created as part of the regular build via a Target dependency in `Engine/Engine.csproj`.
+
+If you see compilation errors related to resources, you can explicitly create the resource files as follows:
+
+```
+PS> Set-Location ResGen
+PS> dotnet run
+```
+
+This will create compilable C# binding files in respective `gen` directory, which will then be used during build.
+
+## Editing `.resx` files
+
+**Do not edit** `.resx` files from Visual Studio. 
+It will try to create `.cs` files for you and you will get whole bunch of hard-to-understand errors.
```

Comment:
I don't have this problem and I am using the latest update of VS. The only problem that I have though is that I have to build twice the first time in a clean checkout for some reason...

Reply:
I took this from the PSCore documents. Does the *.Designer.cs file not get created? If not, I can remove this bit, but if the file is created, that's a problem.

Reply:
My VS (latest version 15.6.1 on Win10) does not create such files any more in your branch, maybe older versions used to do that? Maybe it is better to just put a more generic disclaimer text in there saying that it is recommended to use the latest version that one might need to build twice the first time (to me this seems to be a bug that VS does not wait for the pre-build target to finish)